### ISSUE TITLE
Create Mockups Folder

### ISSUE THREAD

**Issue by mehmetsefa2644** [bounswe/bounswe2017group1#84]
We should create mockups folder in the repo and push mockup files to the folder. So that we won't lose them again.

**Comment by mehmetsefa2644** [comment]
Dear friends, pics of the mockups will be in the pics folder.
We will create a mockups folder, and push the original balsamiq files to the mockups folder. So that we can make changes afterwards.
![bitmoji](https://render.bitstrips.com/v2/cpanel/10212734-322184509_2-s1-v1.png?transparent=1&palette=1&width=246)

**Comment by mehmetsefa2644** [comment]
I created mockups folder, @mehmetgulsen and @mustafafeyzioglu , please rename your .bmpr file and add .bmpr file to mockups folder.

**Comment by mehmetgulsen** [comment]
Good idea. I have uploaded my .bmpr file. You can close this issue once you upload yours @mustafafeyzioglu

**Comment by mustafafeyzioglu** [comment]
Done.
